contracts/governance/FeeSharingProxy/FeeSharingLogic.sol

```diff
@@ -343,6 +401,53 @@ contract FeeSharingLogic is SafeMath96, IFeeSharingProxy, Ownable, FeeSharingPro
 			"FeeSharingProxy::_getTotalVoluntaryWeightedStake: vested stake exceeds total stake"
 		);
 	}
+
+	/**
+	 * @dev Whitelisting converter address.
+	 *
+	 * @param converterAddress converter address to be whitelisted.
+	 */
+	function addWhitelistedConverterAddress(address converterAddress) external onlyOwner {
+		require(Address.isContract(converterAddress), "Non contract address given");
+		require(converterAddress != address(0), "ERR_ZERO_ADDRESS");
+		require(!isWhitelistedConverter[converterAddress], "WHITELISTED_CONVERTER");
+		isWhitelistedConverter[converterAddress] = true;
+		emit WhitelistedConverter(msg.sender, converterAddress);
+	}
+
+	/**
+	 * @dev Removing converter address from whitelist.
+	 *
+	 * @param converterAddress converter address to be removed from whitelist.
+	 */
+	function removeWhitelistedConverterAddress(address converterAddress) external onlyOwner {
+		require(isWhitelistedConverter[converterAddress], "UNWHITELISTED_CONVERTER");
```

Comment:
misleading require msg
```suggestion
		require(isWhitelistedConverter[converterAddress], "NOT_WHITELISTED_CONVERTER");
```

Reply:
This PR has been merged into the PR #385 .
This fix moved to  https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/385/commits/65474a6fdc5ba16dff35acf64a0c5e48b525d930